doc/how_to/waveform_extractor_to_sorting_analyzer.rst

```diff
@@ -0,0 +1,572 @@
+
+Our new class is called the ``SortingAnalyzer``. You can think of this as combining a recording and a sorting into one. There are then lots of ``extensions`` which you can calculate. Waveforms are one extension of the SortingAnalyzer, but aren't special. We hope the new class is simpler to understand, especially for beginners. It also comes with some new features: you can calculate multiple extensions at the same time, it has better caching and better zarr support (hopefully leading to fast interactive widgets in the future).
+
+Convert a WaveformExtractor folder to a SortingAnalyzer folder
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are several backward compatibility tools to help deal with existing ``WaveformExtractor`` folders.
+For now, you can still read these folders using ``load_waveforms``, which creates a ``MockWaveformExtractor`` object.
+This object contains a ``SortingAnalyzer`` which can be accessed and then saved as follows
+
+.. code-block:: python
+
+    waveform_folder_path = "path_to/my_waveform_extractor_folder"
+    new_sorting_analyzer_path = "path_to/my_new_sorting_analyzer_folder"
+
+    extractor = load_waveform(folder = waveform_folder_path)
+    sorting_analyzer = extractor.sorting_analyzer
+    sorting_analyzer.save_as(folder = new_sorting_analyzer_path, format = "binary_folder")
+
+
+The above code creates a ``SortingAnalyzer`` folder at ``new_sorting_analyzer_path``.
+
+Dictionary between SortingAnalyzer and WaveformExtractor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section provides a dictionary for code, to help translate between a ``SortingAnalyzer``
+and a ``WaveformExtractor``, so that users can easily update old code. If you want to learn
+how to use ``SortingAnalyzer`` from scratch, the
+`Get Started <https://spikeinterface.readthedocs.io/en/latest/how_to/get_started.html>`_ guide
+and the `PostProcessing module documentation <https://spikeinterface.readthedocs.io/en/latest/modules/postprocessing.html>`_
+are better places to start.
+
+This section is split into four subsections:
+
+* `Create, load and save <#id2>`_
+* `Checking basic properties <#id3>`_
+* `Compute Extensions <#id4>`_
+* `Quality Metrics <#id5>`_
+
+Throughout this section, we assume that all functions have been imported into the namespace.
+E.g. we have run code such as ``from spikeinterface.postprocessing import extract_waveforms`` for each function. If you have imported
```

Comment:
Where did you find this? I don't think this exists? 

```python
ImportError: cannot import name 'extract_waveforms' from 'spikeinterface.postprocessing
```

Reply:
Good spot - it exists in the pre-SortingAnalyzer spikeinterface versions. Best not to advertise them!